lib/phone-format-interface.js

```diff
@@ -116,7 +116,19 @@ function isValidNumber(phone, country) {
         var phone = cleanPhone(phone);
         var phoneUtil = i18n.phonenumbers.PhoneNumberUtil.getInstance();
         var number = phoneUtil.parseAndKeepRawInput(phone, country);
-        return phoneUtil.isValidNumber(number);
+        if (typeof type !== 'undefined') {
```

Comment:
I'm not sure if this is a real problem, b/c user will have to pass a `null` as a parameter, but probably will be better to use `if (typeof type === 'string') {`.